### ISSUE TITLE
Trying to Match the metrics given in the paper

### ISSUE THREAD

**Issue by BardOfCodes** [mrharicot/monodepth#85]
In the paper, published in [arxiv](https://arxiv.org/pdf/1609.03677.pdf), metrics have been given for what to expect from monodepth model with different training regimes and post-processing.

Correct me if I am wrong, **Ours, No LR**  is the row for VGG model, trained on KITTI, tested on Eigen test set, with no post-processing.
I tried to replicate this performance.
I used the following variables in the code instructions listed in this repository.
for `monodepth_main.py`
checkpoint_file : model_kitti
filenames_file: ./utils/filenames/eigen_test_files.txt
for `evaluate_kitti.py`
split: eigen
predicted_disp_path: disparities.npy

I have used the pretrained models provided. However, the metrics dont match:

<table>
  <tr>
    <td> VGG-weight</td>
    <td>Data split</td>
    <td>abs_rel</td>
    <td>sq_rel</td>
    <td>rms</td>
    <td>log_rms</td>
    <td>a1</td>
    <td>a2</td>
    <td> a3</td>
  </tr>
  <tr>
    <td>** Ours No LR**</td>
    <td>eigen test</td>
    <td>0.152</td>
    <td>1.528</td>
    <td>6.098</td>
    <td>0.252</td>
    <td>0.801</td>
    <td>0.922</td>
    <td>0.963</td>
  </tr>
  <tr>
    <td>model_eigen</td>
    <td>eigen_test</td>
    <td>0.2153</td>
    <td>4.1374</td>
    <td>7.718</td>
    <td>0.315</td>
    <td>0.772</td>
    <td>0.899</td>
    <td>0.947</td>
  </tr>
  <tr>
    <td>model_kitti</td>
    <td>eigen_test</td>
    <td>0.1754</td>
    <td>3.7235</td>
    <td>7.232</td>
    <td>0.293</td>
    <td>0.826</td>
    <td> 0.915</td>
    <td>0.954</td>
  </tr>
</table>

I would like to know if I should try something else to replicate the results.
Thank you!

**Comment by mrharicot** [comment]
Hi,

In table 1 and 2, we test all of our models on the KITTI dataset.
We do use different test data and training splits though.
Table 1 is only on the KITTI split, so it corresponds to the **kitti** models.
Table 2 is only on the EIGEN split, so it corresponds to the **eigen** models.

In Table 1, we test on the 200 ground truth disparities provided by the KITTI stereo 2015 dataset.
In Table 2, we test on 697 reprojected lidar point clouds, following the Eigen et al. paper. This isn't as precise as it ignores occlusions and does not account for motion of both car capturing and the other vehicles/pedestrians.
In Table 2, we evaluate on a crop: so you need to pass this flag `--garg_crop`, except for the last two rows.

Finally noLR is not wihtout post processing but without the left-right consistency.
If you want to test without post-processing you need to use disparities.npy instead of disparities_pp.npy.

I hope this helps!

**Comment by BardOfCodes** [comment]
Thank you for the help. 
The metrics match once I pass the `--garg_crop` flag.
